src/api/cvc4cpp.cpp

```diff
@@ -3356,6 +3445,31 @@ void Solver::setOption(const std::string& option,
   d_smtEngine->setOption(option, value);
 }
 
+Term Solver::ensureTermSort(const Term& t, const Sort& s) const
```

Comment:
Did you double check if this can be used in other places, too? I'd imagine there are several places.

Reply:
As far as I can tell, the only two places where this is useful for now is tuple construction and parsing decimal numbers. It may be useful for datatypes, too, but I couldn't see a place where we do this conversion. We should definitely keep this in mind for the future, though.